<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="logInTab">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>1751</width>
      <height>911</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ROG Fonts</family>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <property name="documentMode">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="existingUserTab">
     <attribute name="title">
      <string>Existing User</string>
     </attribute>
     <widget class="QLineEdit" name="passwdInput">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>290</y>
        <width>521</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>ROG Fonts</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="userIdInput">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>200</y>
        <width>521</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>ROG Fonts</family>
        <pointsize>15</pointsize>
       </font>
      </property>
     </widget>
     <widget class="QLabel" name="passwdLabel">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>280</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>ROG Fonts</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Password               :</string>
      </property>
     </widget>
     <widget class="QLabel" name="userIdLabel">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>200</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>ROG Fonts</family>
        <pointsize>15</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>User Id                      :</string>
      </property>
     </widget>
     <widget class="QLabel" name="checkLogInLabel">
      <property name="geometry">
       <rect>
        <x>760</x>
        <y>370</y>
        <width>861</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>ROG Fonts</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="checkUserIdLabel">
      <property name="geometry">
       <rect>
        <x>1310</x>
        <y>200</y>
        <width>421</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>ROG Fonts</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="checkPasswdLabel">
      <property name="geometry">
       <rect>
        <x>1310</x>
        <y>290</y>
        <width>421</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>ROG Fonts</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="logInButton">
      <property name="geometry">
       <rect>
        <x>1120</x>
        <y>430</y>
        <width>151</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>ROG Fonts</family>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Log In</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="newUserTab">
     <attribute name="title">
      <string>New User</string>
     </attribute>
     <widget class="QLineEdit" name="newPasswdInput">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>540</y>
        <width>521</width>
        <height>61</height>
       </rect>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
     </widget>
     <widget class="QLineEdit" name="newUserIdInput">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>450</y>
        <width>521</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="newPasswdLabel">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>530</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>ROG Fonts</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Password      :</string>
      </property>
     </widget>
     <widget class="QLabel" name="newUserIdLabel">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>450</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>ROG Fonts</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>User Id            :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="rollNoInput">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>360</y>
        <width>521</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="nameInput">
      <property name="geometry">
       <rect>
        <x>720</x>
        <y>270</y>
        <width>521</width>
        <height>61</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="nameLabel">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>270</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>ROG Fonts</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Name                 :</string>
      </property>
     </widget>
     <widget class="QLabel" name="rollNoLabel">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>350</y>
        <width>331</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>ROG Fonts</family>
        <pointsize>20</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Roll No           :</string>
      </property>
     </widget>
     <widget class="QPushButton" name="createAccButton">
      <property name="geometry">
       <rect>
        <x>1030</x>
        <y>620</y>
        <width>201</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>ROG Fonts</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Create Account</string>
      </property>
     </widget>
     <widget class="QLabel" name="checkCreationLabel">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>630</y>
        <width>621</width>
        <height>61</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="dateLabel">
    <property name="geometry">
     <rect>
      <x>1530</x>
      <y>100</y>
      <width>111</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ROG Fonts</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>DATE:</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeLabel">
    <property name="geometry">
     <rect>
      <x>1260</x>
      <y>100</y>
      <width>91</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ROG Fonts</family>
      <pointsize>15</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>TIME:</string>
    </property>
   </widget>
   <widget class="QLabel" name="timeOutputLabel">
    <property name="geometry">
     <rect>
      <x>1360</x>
      <y>100</y>
      <width>141</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ROG Fonts</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="dateOutputLabel">
    <property name="geometry">
     <rect>
      <x>1630</x>
      <y>100</y>
      <width>191</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>ROG Fonts</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
